src/pages/edit-profile/EditProfile.tsx

```diff
@@ -66,6 +68,10 @@ const EditProfile = () => {
   }
 
   const cooperationContent = activeTab && tabsData[activeTab]?.content
+  const errorTooltipHolders = {
+    [UserProfileTabsEnum.Profile]: !tabValidityStatus.profileTab,
+    [UserProfileTabsEnum.ProfessionalInfo]: false // false - doesn't have errors by default
```

Comment:
```suggestion
   [UserProfileTabsEnum.ProfessionalInfo]: !tabValidityStatus.professionalInfoTab
```

Your solution is also good, but what do you think about this implementation? In my opinion, it would be more advantageous for future implementations, as it eliminates the need for a mocked `false` value and enhances maintainability.

Reply:
Good call, thanks